exchange/exchangetest/gdpr-geo-usa.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Consider renaming to `gdpr-geo-usa-off` to mirror `gdpr-geo-usa-on`.

Reply:
Done